src/Database/LSMTree/Internal/Page.hs

```diff
@@ -0,0 +1,63 @@
+-- | Utilities related to pages.
+module Database.LSMTree.Internal.Page (
+    PageNo (..)
+  , nextPageNo
+  , NumPages (..)
+  , getNumPages
+  , PageSpan (..)
+  , singlePage
+  , multiPage
+  , pageSpanSize
+  ) where
+
+import           Control.DeepSeq (NFData (..))
+
+-- | A 0-based number identifying a disk page.
+newtype PageNo = PageNo { unPageNo :: Int }
+  deriving stock (Show, Eq, Ord)
+  deriving newtype NFData
+
+-- | Increment the page number.
+--
+-- Note: This does not encure that the incremented page number exists within a given page span.
```

Comment:
```suggestion
-- Note: This does not ensure that the incremented page number exists within a given page span.
```

Reply:
That’s funny: Everything after the import list in this new module was taken verbatim from the compact-index module; so also the “encure” had been there before. I think at some point I saw this “encure” and just thought that it was some high-level word, which I just didn’t happen to know. 😅 I just had a web search performed for “encure” and found out that apparently “Encure” is just the name of some band. 😁

Reply:
Fixed in 8e5793e5ee0a8fb322b821722ea9493b938b5bae.